### PR TITLE
Implement UI for tx index: new/edit oracle script message

### PR DESCRIPTION
**Pull request by Benzbeeb**
[Example New Oracle Script](https://d3n-scan-pr-804.onrender.com/tx/1A2C74BCF2A7C74CF18679F0179FCF52EA188A0D7C270F19E9EE965AB59672CE)
[Example Edit Oracle Script](https://d3n-scan-pr-804.onrender.com/tx/7510ba23217e7051d20914ebdafad839064eb3c52f65feea4da9b8e8e5c3a89c)

### CODE REVIEW

**Review by evilpeach — DISMISSED** [review]
LGTM!

**Review by prin-r — CHANGES_REQUESTED** [review]
https://ibb.co/Y03kt97
I think for the "DETAIL" we can some how show full address (don't have to ellipsis)

File: scan/src/components/TxIndexPageTable.re [hunk 1]
```diff
@@ -339,13 +339,111 @@ let renderEditDataSource = msg => {
...
+        </div>
+        <VSpacing size=Spacing.sm />
+        <div className={Styles.badge(Colors.pink1)}>
+          <TypeID.OracleScript id={ID.OracleScript.ID(123)} />
+        </div>
+      </div>
+    </Col>
+    <Col size=0.6 alignSelf=Col.Start>
+      <VSpacing size=Spacing.md />
+      <div className={Styles.addressContainer(170)}>
```
Comment (prin-r) [inline scan/src/components/TxIndexPageTable.re]: 170  is too small to be able to show full address

**Review by prin-r — COMMENTED** [review]
Screen Shot 2563-03-13 at 19 42 53
Use AddressRender component for this. It should have ellipsis and link to account index by default.

**Review by prin-r — CHANGES_REQUESTED** [review]
Screen Shot 2563-03-13 at 19 47 57
We have enough space to show full address. Don't have to be ellipsis

File: scan/src/components/TxIndexPageTable.re (hunk 1 above)
Comment (prin-r) [reply scan/src/components/TxIndexPageTable.re]: I try 300 via chrome and still got ellipsis. So I try 500 and ellipsis is gone.

**Review by prin-r — COMMENTED** [review]
Screen Shot 2563-03-13 at 19 47 57
We have enough space to show full address. Don't have to be ellipsis

**Review by taobun — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**